packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.styles.ts

```diff
@@ -12,59 +12,70 @@ import {
   getStyles as getSplitButtonStyles
 } from '../SplitButton/SplitButton.styles';
 
-const DEFAULT_BUTTON_HEIGHT = '40px';
-const DEFAULT_PADDING = '0 4px';
+const COMMAND_BUTTON_HEIGHT = '40px';
```

Comment:
Yeah, I like the idea of removing any height (in case user wants to put it into smaller navigation) it just leaves the example looking kinda funky. Also, should I add align-self: stretch so that they'll automatically stretch to fit flex container? Or assume the user will add that?